app/views/products/show.html.erb

```diff
@@ -1,81 +1,123 @@
-<p style="color: green" class="text-center mt-3"><%= notice %></p>
-<%= form_tag add_to_cart_product_path(@product), method: :post do %>
-  <%= submit_tag 'Add to Cart', class: 'btn btn-primary' %>
-<% end %>
 <div class="container">
   <div class="row text-center">
-    <h1 class="my-5 login-heading mb-4 fw-medium fs-1 fw-light">Product Details</h1>
+    <% if current_user && current_user.usertype == 'admin' %>
+      <!-- Admin content-->
+      <h1 class="my-5 login-heading mb-4 fw-medium fs-1 fw-light">Product Details</h1>
+      <div class="row col-10 offset-1">
+        <table class="table table-bordered text-center">
+          <thead>
+            <tr>
+              <th>Attribute</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td class="align-middle">Image</td>
+              <td>
+                <% if @product.image.attached? %>
+                  <%= image_tag rails_blob_path(@product.image, disposition: "attachment", only_path: true), class: "img-thumbnail img-fluid", width: "100", height: "100" %>
+                <% else %>
+                  No Image
```

Comment:
Enhancing the user experience involves adding a default system image rather than using plain text.